docs/extensions.md

```diff
@@ -1,235 +0,0 @@
-
-```
-git clone https://github.com/owncloud/phoenix.git
-cd phoenix
-
-yarn install
-yarn dist
-```
-
-We can tell ocis to use the compiled assets:
-
-Kill `ocis phoenix`, then use the compiled assets when starting phoenix.
-
-```console
-cd ../ocis
-PHOENIX_ASSET_PATH="`pwd`/../phoenix/dist" bin/ocis phoenix
-```
-
-## The ownCloud design system
-
-The owncloud design system contains a set of ownCloud vue components for phoenix or your own ocis extensions. Use it for a consistent look and feel.
-
-Point your browser to https://owncloud.github.io/owncloud-design-system and check the available components. Live editing the examples in the browser is supported.
-
-note: There is a bug with navigation sub items: either click a nav item twice or refresh the page
-
-## External phoenix apps
-
-This is what hello is: copy and extend!
-
-1. Phoenix is configured using the config.json which is served by the phoenix service (either `bin/ocis phoenix` or `bin/ocis-phoenix server`)
-
-2. point ocis phoenix to the web config which you extended with an external app:
-`PHOENIX_WEB_CONFIG="`pwd`/../phoenix/config.json" PHOENIX_ASSET_PATH="`pwd`/../phoenix/dist" bin/ocis phoenix`
-
-```json
-{
-  "server": "http://localhost:9140",
-  "theme": "owncloud",
-  "version": "0.1.0",
-  "openIdConnect": {
-    "metadata_url": "http://localhost:9140/.well-known/openid-configuration",
-    "authority": "http://localhost:9140",
-    "client_id": "phoenix",
-    "response_type": "code",
-    "scope": "openid profile email"
-  },
-  "apps": [],
-  "external_apps": [
-    {
-      "id": "hello",
-      "path": "http://localhost:9105/hello.js",
-      "config": {
-        "url": "http://localhost:9105"
-      }
-    },
-    {
-      "id": "myapp",
-      "path": "http://localhost:6789/superapp.js",
-      "config": {
-        "backend": "http://someserver:1234",
-        "myconfig": "is awesome"
-      }
-    }
-  ]
-}
-```
-
-## Phoenix extension points
-
-> Note: For an up to date list check out [the phoenix documentation](https://github.com/owncloud/phoenix/issues/2423). 
-
-Several ones available:
-
-### Phoenix core
-- App switcher (defined in config.json)
-- App container (loads UI of your extension)
-
-### Files app
-- File action
-- Create new file action
-- Sidebar
-- Quick access for sidebar inside of file actions (in the file row)
-
-Example of a file action in the `app.js`:
-```js
-const appInfo = {
-  name: 'MarkdownEditor',
-  id: 'markdown-editor',
-  icon: 'text',
-  isFileEditor: true,
-  extensions: [{
-    extension: 'txt',
-    newFileMenu: {
-      menuTitle ($gettext) {
-        return $gettext('Create new plain text file…')
-      }
-    }
-  },
-  {
-    extension: 'md',
-    newFileMenu: {
-      menuTitle ($gettext) {
-        return $gettext('Create new mark-down file…')
-      }
-    }
-  }]
-}
-```
-
-For the side bar have a look at the files app, `defaults.js` & `fileSideBars`
-
-## API driven development
```

Comment:
I am not sure where to include this part. This seems more general information than extensions specific. 🤷